src/Session/commands.py

```diff
@@ -51,8 +51,35 @@ def add_food(self):
             print(ADD_FOOD_ERROR_MESSAGE)
             print(SUGGESTED_ACTIONS_MESSAGE.format(listInQuotes([OPEN_FRIDGE, OPEN_FREEZER, CREATE_FREEZER, CREATE_FRIDGE])))
 
+    ''' Prompts user to remove foods when they are in an open food storage. If the user is not in an opened
+        food storage, suggests the user opens an existing food storage or create a new one. '''
     def remove_food(self):
-        pass
+        # Profile check
+        if not self.profile:
+            print(NO_LOADED_PROFILE_MESSAGE, \
+                SUGGESTED_ACTIONS_MESSAGE.format(listInQuotes([CREATE_PROFILE, LOAD])))
+            return
+        
+        # Check if food storage is open
+        if self.profile.getOpenFoodStorage():
+            food_list = []
+            while True: 
+                food = input(REMOVE_FOOD_NAME_MESSAGE)
+                food_list.append(food)
+                    
+                while True:
+                    remove_more_food = input(REMOVE_MORE_FOOD_MESSAGE)
+                    if remove_more_food == YES:
+                        break
+                    elif remove_more_food == NO:
+                        self.profile.removeFoods(food_list)
+                        print(REMOVE_FOOD_SUCCESS_MESSAGE_FINAL)
```

Comment:
I think you should only return this success message if `removeFoods` is successful (See comments for that method)

Reply:
Same as previous comment.